libvast/test/event.cpp

```diff
@@ -55,7 +55,7 @@ FIXTURE_SCOPE(event_tests, fixture)
 
 TEST(basics) {
   CHECK_EQUAL(e.type().name(), "foo");
-  REQUIRE(caf::holds_alternative<vector>(e.data()));
+  REQUIRE(caf::holds_alternative<std::vector<data>>(e.data()));
```

Comment:
```suggestion
  REQUIRE(caf::holds_alternative<list>(e.data()));
```

Reply:
Here, `list` is the wrong way to think about this. An event is a sequence of `data`, not an instance of `list`. This is why I changed it to `std::vector<data>`. It'll change with the `record` change in `data` as well.

Reply:
I think this is just unnecessarily confusing. A `list` now is exactly a sequence of `data`, because that is quite literally its definition.

Reply:
Yes, but it's a value *for a single field*. Here, we are modeling a *sequence of values*, each of which represent a single field.

Reply:
I still disagree with you, but not not enough to argue about this one.

Reply:
Fair enough, it'll change soon anyway once `record` is there.